docs/source/model_design/programming_vivarium/overview_engineering_files/index.rst

```diff
@@ -29,12 +29,159 @@ Overview of Engineering Files
 
 .. contents::
    :local:
-   :depth: 1
+   :depth: 2
 
-.. todo::
 
-  Add to this page. Include explanations for the relevant engineering files (model specs, branches file, loader.py, etc.) that includes when the functions are called, when you might need to look at them, what information is stored where, and any other relevant information for RT based tasks. 
+This document is designed to provide an overview of the files found in the 
+engineering repository and some examples of when you might need to use 
+each file. This is NOT inclusive of files as some are less relevant 
+for research team members, and the files included change between models. 
 
-  Integrate links to: `YAML basics <https://vivarium.readthedocs.io/en/latest/concepts/model_specification/yaml_basics.html#model-specification-yaml-concept>`_, `model specs file <https://vivarium.readthedocs.io/en/latest/concepts/model_specification/index.html>`_, and the `branches file <https://vivarium-cluster-tools.readthedocs.io/en/latest/branch.html>`_. 
+This page is organized in sections by the different folders found in engineering 
+repositories and then the pages within folders, as relevant. 
 
+Components
+----------
 
+The :code:`components` folder contains files on the information for each 
+modeling "component". This includes things like disease models, 
+risk factors, pregnancies, the health care system, or 
+interventions. 
+
+The files included will be specific to your project rather than generalizable. 
+Additionally, these files will mostly be maintained and written by the 
+engineering team. For those reasons, we won't elaborate on these files here. 
+
+Constants
+---------
+
+The :code:`constants` folder contains input information for the model. This includes 
```

Comment:
The results.py file in this folder is also where we tell process_results what we expect the output to look like (ie observers and stratifications). This file needs to be updated if our outputs change (but we still need to decide who will be doing this).